src/main/java/com/exasol/projectkeeper/validators/dependencies/MavenDependencyReader.java

```diff
@@ -0,0 +1,119 @@
+    }
+
+    /**
+     * Read the dependencies of the pom file (including plugins).
+     * 
+     * @param pomFile pom file to read
+     * @return list of dependencies
+     */
+    public List<com.exasol.projectkeeper.validators.dependencies.Dependency> readDependencies(final File pomFile) {
+        final Model model = parsePomFile(pomFile);
+        return getDependenciesIncludingPlugins(model).map(this::getDependenciesLicense).collect(Collectors.toList());
+    }
+
+    private Stream<Dependency> getDependenciesIncludingPlugins(final Model model) {
+        return Stream.concat(model.getDependencies().stream(),
+                model.getBuild().getPlugins().stream().map(this::convertPluginToDependency));
+    }
+
+    private Dependency convertPluginToDependency(final Plugin plugin) {
+        final Dependency dependency = new Dependency();
+        dependency.setGroupId(plugin.getGroupId());
+        dependency.setArtifactId(plugin.getArtifactId());
+        dependency.setVersion(plugin.getVersion());
+        dependency.setScope("plugin");
+        return dependency;
+    }
+
+    private com.exasol.projectkeeper.validators.dependencies.Dependency getDependenciesLicense(
+            final Dependency dependency) {
+        try {
+            final Model dependenciesPom = this.artifactModelReader.readModel(dependency.getArtifactId(),
+                    dependency.getGroupId(), dependency.getVersion());
+            final List<License> licenses = dependenciesPom.getLicenses().stream()
+                    .map(license -> new License(license.getName(), license.getUrl())).collect(Collectors.toList());
+            return new com.exasol.projectkeeper.validators.dependencies.Dependency(getName(dependenciesPom),
+                    dependenciesPom.getUrl(), licenses, mapScope(dependency.getScope()));
+        } catch (final ProjectBuildingException exception) {
+            throw new IllegalStateException(ExaError.messageBuilder("E-PK-49")
+                    .message("Failed to get license information for dependency {{groupId}}:{{artifactId}}.",
+                            dependency.getGroupId(), dependency.getArtifactId())
+                    .toString(), exception);
+        }
+    }
+
+    private String getName(final Model dependenciesPom) {
+        if (dependenciesPom.getName() == null || dependenciesPom.getName().isBlank()) {
+            return dependenciesPom.getArtifactId();
+        } else {
+            return dependenciesPom.getName();
+        }
+    }
+
+    private Scope mapScope(final String scope) {
+        if (scope == null) {
+            return COMPILE;
+        } else {
+            switch (scope) {
+            case "compile":
+            case "provided":
+            case "system":
+            case "import":
+                return COMPILE;
+            case "test":
+                return TEST;
+            case "runtime":
+                return Scope.RUNTIME;
```

Comment:
```suggestion
                return RUNTIME;
```

Reply:
Maybe a default one will be `compile`? The method will be much compact then

Reply:
I think it helps to have an exception if the scope is unknown. Then we can think about where to put it.

Reply:
I think it's not really nice to have exceptions and be not able to use keeper(or new dependency) at all if the scope is unknown.As a user I would be happy if we change it

Reply:
I discussed with @redcatbear and we decided that itÄs that unlikely to have a new scope that an exception is ok here